src/engine/renderer/glsl_source/computeLight_fp.glsl

```diff
@@ -72,6 +72,18 @@ void computeLight( vec3 lightDir, vec3 normal, vec3 viewDir, vec3 lightColor,
   float NdotH = clamp( dot( normal, H ), 0.0, 1.0 );
 #endif // USE_PHYSICAL_MAPPING || r_specularMapping
 
+  float NdotL = clamp( dot( normal, lightDir ), 0.0, 1.0 );
+
+#if defined(r_halfLambertLighting)
```

Comment:
I think you will get a different result if the USE_PHYSICAL_MAPPING flag is on, and either r_halfLambertLighting or r_wrapAroundLighting is set.

Reply:
Yes it is expected this code enables halfLamberLighting and wrapAroundLighting for PBR too as PBR is just another way to compute specularity and things like that.

When refactoring code at 0.51 time I failed to see how I can also factorize that and just let it for the future (PBR code was already not using those features yet, so there was no regression anyway).